app/build.gradle

```diff
@@ -288,6 +288,10 @@ dependencies {
     testImplementation "io.mockk:mockk:1.10.0"
     // Koin
     testImplementation "org.koin:koin-test:$koin_version"
+
+    // Kronos ntp clock
+    def kronos_latest_version = "0.0.1-alpha09"
```

Comment:
Last dependencies are preserved for the test flavored so let's move this before the test dependencies.

Reply:
dependency line moved